sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample08_Interop.md

```diff
@@ -0,0 +1,52 @@
+## CRUD operations
+
+This sample demonstrates how to interoperate with messages that are sent or received using the `WindowsAzure.ServiceBus` library. The `WindowsAzure.ServiceBus` library uses the DataContractSerializer to serialize the BrokeredMessage body. Because of this, when attempting to interoperate with this library, there a few additional steps that are needed.
+
+### Sending a message using `Azure.Messaging.ServiceBus` that will be received with `WindowsAzure.ServiceBus`
+
+```C# Snippet:ServiceBusInteropSend
+ServiceBusSender sender = client.CreateSender(queueName);
+// When constructing the `DataContractSerializer`, We pass in the type for the model, which can be a strongly typed model or some pre-serialized data.
+// If you use a strongly typed model here, the model properties will be serialized into XML. Since JSON is more commonly used, we will use it in our example, and
+// and specify the type as string, since we will provide a JSON string.
+var serializer = new DataContractSerializer(typeof(string));
+var stream = new MemoryStream();
```

Comment:
```suggestion
var stream = new MemoryStream();
```

I recognize that there's no practical difference,  just sticking with the "if it's disposable, we should demonstrate disposing it" thought.